charts/renku/values.yaml

```diff
@@ -58,20 +66,27 @@ global:
       ## Name of the postgres user to be used to access the Event Log db
       postgresUser: eventlog
       ## Postgres password to be used to access the Event Log db
-      ## Generated using: `openssl rand -hex 32`
       postgresPassword:
+        value:
+        overwriteOnHelmUpgrade: false
+      existingSecret: '{{ template "renku.fullname" . }}-db-postgres'
     tokenRepository:
       ## Name of the postgres user to be used to access the db storing access tokens
       postgresUser: tokenstorage
       ## Postgres password to be used to access the db storing access tokens
-      ## Generated using: `openssl rand -hex 32`
       postgresPassword:
+        value:
+        overwriteOnHelmUpgrade: false
+      existingSecret: '{{ template "renku.fullname" . }}-token-postgres'
     jena:
       ## specify which dataset should be used in Jena
       dataset:
   ## Set to true if using https
   useHTTPS: false
 
+  ## Set to true to recreate secrets (credentials etc.) when updating the chart
```

Comment:
Looks like this is not used as this option can now be set on each secret individually.